js/form/src/Dropdown.svelte

```diff
@@ -23,12 +23,12 @@
 		blur: undefined;
 	}>();
 
-	let inputValue: string,
+	let inputValue: string | undefined,
```

Comment:
This is needed so that the `gr.Dropdown` can handle `null` values properly. Previously, sending `null` to a `Dropdown` would cause the value to be subdued:

<img width="776" alt="image" src="https://github.com/gradio-app/gradio/assets/1778297/68ccea7a-d24a-4124-b207-fbe02b05c73e">